vscode_extension/CHANGELOG.md

```diff
@@ -0,0 +1,29 @@
+# Version history
+
+## 0.3.6
+
```

Comment:
I think we can erase the changelog for earlier versions and just write "- First open source release" under this header.